environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -1454,6 +1454,11 @@ objects:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
+          - name: AUTHORIZE_URL
```

Comment:
this change first needs to go into the telemeter-server jsonnet code

Reply:
i thought i did them in https://github.com/openshift/telemeter/pull/288

Reply:
Yes and they were pulled in with Telemeter as dependency. So this LGTM now.

Reply:
Looks like we missed something there: https://github.com/openshift/telemeter/pull/322